docs/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -143,19 +143,31 @@ Now you can select a hardware package that can cover these requirements. In this
+|                                                                                     | Basic          | Standard       | Advanced       |
+| :---------------------------------------------------------------------------------- | :------------- | :------------- | :------------- |
+| Max Throughput **Tasks/day** **\***                                                 | 4.5 M          | ? M            | ? M            |
+| Max Throughput **Tasks/second** **\***                                              | 52             | ?              | ?              |
+| Max Throughput **Process Instances/day** **\*\***                                   | 2.9 M          | ? M            | ? M            |
+| Max Total Number of Process Instances stored (in Elasticsearch in total) **\*\*\*** | ?              | ?              | ?              |
+| Typical cluster size for licensing **\*\*\*\***                                     | 1x, 2x, 3x, 4x | 1x, 2x, 3x, 4x | 1x, 2x, 3x, 4x |
 
-**\*\*** These are the resource limits configured in the Kubernetes cluster and are always subject to change.
+**\*** Tasks (Service Tasks, Send Tasks, User Tasks, and so on) completed per day is the primary metric, as this is easy to measure and has a strong influence on resource consumption. This number assumes a constant load over the day.
+
+**\*\*** As Tasks are the primary resource driver, the number of Process Instances supported by a cluster is calculated based on the assumption of an average of 10 tasks per process. Customers can calculate a more accurate Process Instance estimate using their anticipated number of tasks per process.
+
+**\*\*\*** Total number of process instances within the retention period, regardless of if they are active or finished. This is limited by disk space, CPU, and memory for running and historical process instances available to ElasticSearch. Calculated assuming a typical set of process variables for process instances. Note that it makes a difference if you add one or two strings (requiring ~ 1kb of space) to your process instances, or if you attach a full JSON document containing 1MB, as this data needs to be stored in various places, influencing memory and disk requirements. If this number increases, you can still retain the runtime throughput, but Tasklist, Operate, and/or Optimize may lag behind.
+
+Data Retention has an influence on the amount of data that is kept for completed instances in your cluster. The default Data retention is set to 30 days, which means that data that is older than 30 days gets removed from Operate and Tasklist. If a process instance is still active, it is fully functioning in runtime, but customers are not able to access historical data older than 30 days from Operate and Tasklist. Data retention is set to 6 months, meaning that data that is older than 6 months will be removed from Optimize. Up to certain limits Data Retention can be adjusted by Camunda on request.
```

Comment:
This could link to the data retention section rather that we have in docs

Reply:
Added link 👍